editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -182,13 +182,22 @@ void HelpBrowser::closeDocument()
 
 void HelpBrowser::gotoHelpFor( const QString & symbol )
 {
-    QString code = QStringLiteral("HelpBrowser.openHelpFor(\"%1\")").arg(symbol);
+    QString escaped{symbol};
+    escaped.replace('\"', "\\\"");
+
+    QString code = QStringLiteral("HelpBrowser.openHelpFor(\"%1\")").arg(escaped);
     sendRequest(code);
 }
 
 void HelpBrowser::gotoHelpForMethod( const QString & className, const QString & methodName )
 {
-    QString code = QStringLiteral("HelpBrowser.openHelpForMethod( %1.findMethod(\\%2) )").arg(className, methodName);
+    QString escapedClass{className};
+    QString escapedMethod{methodName};
+    escapedClass.replace('\"', "\\\"");
+    escapedMethod.replace('\"', "\\\"");
```

Comment:
small amount of duplicated code here -- how about making this into a function?

Reply:
good point, I've done that now. :)